<!-- -->
<launch>
    <arg name="resolution" default="0.1" />
    <arg name="cell_width" default="500" />
    <arg name="cell_height" default="500" />
    <arg name="points_topic" default="points_no_ground" />
<!--    <arg name="points_topic" default="rslidar_points" />-->
    <arg name="offset_x" default="25.0" />
    <arg name="offset_y" default="0.0" />
    <arg name="offset_z" default="-2.0" />


	<node pkg="object_map" type="points2costmap" name="points2costmap" output="screen">
        <param name="resolution" value="$(arg resolution)" />
        <param name="cell_width" value="$(arg cell_width)" />
        <param name="cell_height" value="$(arg cell_height)" />
        <param name="points_topic" value="$(arg points_topic)" />
        <param name="offset_x" value="$(arg offset_x)" />
        <param name="offset_y" value="$(arg offset_y)" />
        <param name="offset_z" value="$(arg offset_z)" />
	</node>

</launch>

